File: TransCoder_tokenized_test_set_functions/java/PRODUCT_NODES_K_TH_LEVEL_TREE_REPRESENTED_STRING.java
static int productAtKthLevel ( String tree, int k ) {
  int level = - 1;
  int product = 1;
  int n = tree . length ( );
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( tree . charAt ( i ) == '(' ) level ++;
    else if ( tree . charAt ( i ) == ')' ) level --;
    else {
      if ( level == k ) product *= ( tree . charAt ( i ) - '0' );
    }
  }
  return product;
}

File: results/py/PRODUCT_NODES_K_TH_LEVEL_TREE_REPRESENTED_STRING.cpp
int f_filled ( string tree , int k ) {
  int level = - 1 ;
  int product = 1 ;
  int n = tree . length ( ) ;
  for ( int i = 0 ;
  i <= n ;
  i ++ ) {
    if ( ( tree [ i ] == '(' ) || ( tree [ i ] == ')' ) ) level ++ ;
    else if ( ( tree [ i ] == '(' ) || ( tree [ i ] == ')' ) ) level -- ;
    else {
      if ( ( level == k ) && ( i == 0 ) ) product *= ( atoi ( tree [ i ] . c_str ( ) ) - atoi ( "0" ) ) ;
    }
  }
  return product ;
}
